iphone/Classes/TiUIWebViewProxy.m

```diff
@@ -14,7 +14,9 @@
 #import "TiNetworkCookieProxy.h"
 #import "TiUIWebView.h"
 
-@implementation TiUIWebViewProxy
+@implementation TiUIWebViewProxy {
+  BOOL _hasAddedTisdkScript;
+}
```

Comment:
Ok, almost there! Use a private property in a class extension in the .m file, not a raw ivar. This gives you clear encapsulation, dot-syntax, and proper semantics under ARC/KVC. That's basically the modern Objective-C way to do this.

```objc
@interface TiUIWebViewProxy ()
@property (nonatomic, assign) BOOL hasAddedTisdkScript;
@end

@implementation TiUIWebViewProxy
```